ash/src/vk/features.rs

```diff
@@ -19,31 +19,22 @@ impl ::std::clone::Clone for StaticFn {
     }
 }
 impl StaticFn {
-    pub fn load<F>(mut _f: F) -> Self
+    pub fn load<F>(mut _f: F) -> std::result::Result<Self, MissingEntryPoint>
```

Comment:
Careful, this file is autogenerated!

https://github.com/MaikKlein/ash/blob/b3a010a3154d0cd9d4943e794a0e90916da93936/generator/src/lib.rs#L2210-L2217

We could possibly disable generating `fn load` and implement that manually elsewhere.

Reply:
Ah, I wasn't aware. Maybe a good idea to put a big comment at the top of the file for future editors.

I don't really trust myself to go messing with the generator just yet, but there is perhaps an alternative. Instead of letting `StaticFn` do the check, it could be done by `Entry`, and leave `StaticFn` as it is. What do you think?